app/src/main/java/com/example/mylittlelibrary/ui/activities/ListActivity.kt

```diff
@@ -26,7 +26,7 @@ class ListActivity : AppCompatActivity() {
         binding = ActivityListBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        var intent = Intent(this, ListActivity::class.java)
+        val intent = Intent(this, ListActivity::class.java)
```

Comment:
The intent variable can be declared with val instead of var because the intent is not modified after being declared, which was a suggested change made by Android Studio.

Reply:
Good idea.  Assume val by default.  Use var only if you know it will be changed.